### PR TITLE
Update component docs

### PR DESCRIPTION
**Pull request by captainsafia**
Updates the config and adds an introduction paragraph.

### CODE REVIEW

File: doc/components.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Welcome to the nteract components documentation! One this web page, you'll find examples of how to use the React components within the nteract ecosystem to build your own custom user interfaces. These docs will showcase best practices, example implementations, and provide details on how you can use different React components together to build your own interactive experiences.
```
Comment (rgbkrk) [inline doc/components.md]: "On this web page,"

Comment (rgbkrk) [reply doc/components.md]: I realize that I lean heavily on the phrase "interactive experiences" which is kind of meaningless to frontend developers "isn't that what I'm always building?". Maybe we should end on "build your own notebook, console, or your own unique interactive computing experience".

Comment (kelleyblackmore) [reply doc/components.md]:
```suggestion
Welcome to the nteract components documentation! On this web page, you'll find examples of how to use the React components within the nteract ecosystem to build your own custom user interfaces. These docs will showcase best practices, example implementations, and provide details on how you can use different React components together to build your own interactive experiences.
```

Comment (kelleyblackmore) [reply doc/components.md]: Wanted to try out the "Suggested Change" feature here ^^

**Review by rgbkrk — APPROVED** [review]

**Pull request merged**